notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb

```diff
@@ -1170,11 +1106,11 @@
         "id": "e1d089d1d0d9"
       },
       "source": [
-        "### Get the latest state of the production endpoint\n",
+        "## Get the latest state of the production endpoint\n",
         "\n",
-        "Now that the pipeline has finished, the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
+        "Once your pipeline execution is finished, you can notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
```

Comment:
change: "Once your pipeline execution is finished, you can notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint"

to: "Once your pipeline execution is finished, notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint"